label-pull-requests/main.mjs

```diff
@@ -46,6 +46,8 @@ async function main() {
 
         // Extend every file object with its content
         for (const file of files.data) {
+            file.patch ||= ""
+
```

Comment:
Would be nice if callers didn't need to include `^[+-]` since they always want to look at changed lines (otherwise they would just use `contents`) and instead handle that here, what do you think? Maybe something like

```suggestion
            file.patch ||= ""
            file.patch = file.patch
                .split("\n")
                .filter(line => /^[-+]/.test(line))
                .join("\n");
```